Library/Homebrew/cask/test/Casks/compliance_test.rb

```diff
@@ -2,13 +2,15 @@
 
 describe "Casks" do
   with_environment "HOMEBREW_DEVELOPER" => nil do
-    Hbc.all.reject {|c| c.is_a?(Hbc::TestCask) }.each do |cask|
-      describe "#{cask}" do
-        it "passes audit" do
-          audit = Hbc::Audit.new(cask)
-          audit.run!
-          audit.errors.must_equal [], "[#{cask}] Cask audit must be error free"
-          audit.warnings.must_equal [], "[#{cask}] Cask audit must be warning free"
+    shutup do
```

Comment:
Done in https://github.com/Homebrew/brew/pull/1329/commits/fb9ca2c2d390624633583f26a4600d85e2a5cae1.